vtex-apps/store-components
# Push 'newsletterInput' pixel event after newsletter mount

#### What problem is this solving?
Pixel listrak try to find newsletter input, but the element isn't rendered when the listrak script is loaded, so the pixel doesn't work properly.

#### How to test it?
[Workspace](https://brasileirovtex--eriksbikeshop.myvtex.com/)

1. Open the store page
2. Go to newsletter field
3. add an email and submit
4. check if a request is made to https://sca1.listrakbi.com/ 
![image](https://user-images.githubusercontent.com/15680320/109345422-e03fec80-784e-11eb-91fb-db704e9246b2.png)

#### Describe alternatives you've considered, if any.
Change render-runtime, but seems overkill. 

#### Related to / Depends on

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/8rEiN2GsOdQTmFV38t/giphy.gif)
